skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/SakBrev.kt

```diff
@@ -184,6 +185,13 @@ fun Route.sakBrev(dto2ApiService: Dto2ApiService, brevredigeringService: Brevred
 
             }
         }
+
+        route("/alltidValgbareVedlegg") {
```

Comment:
`brevId`-path-parameter er ikke inkludert i den ruten du har satt opp her.
Jeg ville gått for `get("{brevId}/alltidValgbareVedlegg") { ... }`. (og bare droppet `route`-wrapping siden det kun er et endepunkt).